csharp/protos/unittest_issues.proto

```diff
@@ -137,4 +139,11 @@ message OneofMerging {
     string text = 1;
     Nested nested = 2;
   }
-}
\ No newline at end of file
+}
+
+message NullValueOutsideStruct {
+  oneof value {
+    string string_value = 1;
+    google.protobuf.NullValue null_value = 2;
+  }
+}
```

Comment:
Also add the test when google.protobuf.NullValue is a normal field(not oneof field), and make sure nothing is printed in this situation

Reply:
Will do - although presumably it *should* be formatted when FormatDefaultValues is set to true... at which point should it format as `null` or `"NULL_VALUE"`? I'd assume `null`.

Reply:
Done - see the second commit in the PR.